headless/src/jsMain/kotlin/dev/fritz2/headless/foundation/OpenClose.kt

```diff
@@ -48,7 +48,7 @@ abstract class OpenClose: WithJob {
         return if (domNode is HTMLButtonElement) {
             clicks
         } else {
-            keydowns.filter { shortcutOf(it) in listOf(Keys.Space, Keys.Enter) }
+            merge(clicks, keydowns.filter { shortcutOf(it) in listOf(Keys.Space, Keys.Enter) })
```

Comment:
I would like to change this to `setOf` -> a set is the idiomatic type for containment checks

Reply:
Will do